lib/init/grass.py

```diff
@@ -92,7 +92,7 @@
 if "GISBASE" in os.environ and len(os.getenv("GISBASE")) > 0:
     GISBASE = os.path.normpath(os.environ["GISBASE"])
 else:
-    GISBASE = os.path.normpath("@GISBASE@")
+    GISBASE = os.path.normpath("@SET_GISBASE@")
```

Comment:
If we are renaming the variable, perhaps we can use a more descriptive name.

Reply:
I need to check why someone (was it @Mahesh1998?) renamed it.

Reply:
It was never SET_GISBASE as far as I can tell and it is not now. It is also unclear to me what "SET" means here. If you can revert to GISBASE and everything still works, I would go with that for this PR.

Reply:
This originates from me with c2e2ebb4417ebc73bb8f309a4340bb6bbdb74d04  and 7a3b557518cbd40e6fd03c56b8fe08dfa7e7576e. It is possible this was a temp fix, which grew...